addons/website_sale/static/src/js/product_configurator_dialog/product_configurator_dialog.js

```diff
@@ -59,4 +56,14 @@ patch(ProductConfiguratorDialog.prototype, {
     showShopButtons() {
         return this.props.isFrontend && !this.props.edit;
     },
+
+    /**
+     * Computes and returns the formatted total price of all selected products.
+     *
+     * @return {String} - Localized string representing the total.
+     */
+    get formattedTotal() {
+        const total = this.state.products.reduce((sum, p) => sum + p.price * p.quantity, 0);
+        return `${_t('Total:')} ${formatCurrency(total, this.env.currency.id)}`;
+    },
```

Comment:
Why we need this method? I guess it does not take all the things into account like combo products and all instead you should just use totalMessage method I guess